src/Logic/UiUtil.cs

```diff
@@ -463,31 +463,35 @@ private static Font DefaultSystemFont
                     font = new Font(Configuration.DefaultLinuxFontName, 8F);
                 }
 
-                try
+                if (IsFontPresent(font.Name))
                 {
-                    // Bold + italic + regular must be present
-                    font = new Font(font.Name, 9, FontStyle.Bold);
-                    font = new Font(font.Name, 9, FontStyle.Italic);
-                    font = new Font(font.Name, 9, FontStyle.Regular);
+                    _defaultSystemFont = font;
                 }
-                catch
+                else if (IsFontPresent(SystemFonts.DefaultFont.Name)) // system default

```

Comment:
I don't think we need to check if SystemFonts.DefaultFont exists on the current system...

Reply:
yes!